week-1/G-strings-methods/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `name`
-
-console.log(name);
+var name = "mariama";
+var namelength = name.length;
+console.log(
+  "my name is " + name + " and my name is " + namelength + " characters long"
+);
```

Comment:
you should have two console log 
First message:
My name is (your name) and my name is (name length) characters long
second message:
My name is (use variable name (line 2) and you need to use trim method to remove space) and my name is (name length) characters long